cmd/serve.go

```diff
@@ -12,15 +14,29 @@ var serveCmd = &cobra.Command{
 	Use:   "serve",
 	Short: "Starts the HTTP REST APIs server",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		webBuilder := new(config.WebBuilder).InitFromViper(viper.GetViper())
-		server := new(web.Server).InitFromWebBuilder(webBuilder)
+		mailChannel := make(chan mail.Mail)
+
+		builder := new(config.WebBuilder).InitFromViper(viper.GetViper())
+
+		resources.Outbox = mailChannel
```

Comment:
Please, create a function to initialize the resource package with the needed dependencies

Reply:
ok